test/io/jhh/readbinary-error.chpl

```diff
@@ -0,0 +1,8 @@
+use IO;
```

Comment:
I mentioned it on the definition of one of the config params, but I think you want to delete this test.

Reply:
I want to test that `readBinary` works correctly when an I/O error occurs.